control-plane/csi-driver/src/context.rs

```diff
@@ -414,6 +425,31 @@ impl TryFrom<&HashMap<String, String>> for PublishParams {
     }
 }
 
+/// Validate the topology parameters.
+pub(crate) fn validate_single_topology_param(
+    pool_affinity_topology_label: Option<HashMap<String, String>>,
+    pool_affinity_topology_key: Option<Vec<String>>,
+    pool_has_topology_key: Option<HashMap<String, String>>,
+    node_affinity_topology_label: Option<HashMap<String, String>>,
+    node_affinity_topology_key: Option<Vec<String>>,
+    node_has_topology_key: Option<HashMap<String, String>>,
+    node_spread_topology_key: Option<HashMap<String, String>>,
+) -> Result<(), tonic::Status> {
+    if pool_affinity_topology_label.is_some()
```

Comment:
Am I missing something here, this is just checking if any option is enabled not if more than one are enabled?

Reply:
Thanks for poiniting this out, rectified it.